addons/purchase_stock/tests/test_replenish_wizard.py

```diff
@@ -318,6 +293,7 @@ def test_supplier_delay(self):
             'route_ids': [Command.link(self.route_buy.id)],
         })
         vendor1 = self.env['res.partner'].create({'name': 'vendor1', 'email': 'from.test@example.com'})
+        vendor2 = self.env['res.partner'].create({'name': 'vendor2', 'email': 'from.test2@example.com'})
```

Comment:
Create both vendor in batch, same for supplierinfo

Reply:
Don't mark as resolved not done changes (I think you did the change on other lines but not here)